src/tableHeaderCells.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Table } from 'evergreen-ui'
+import styled from 'styled-components'
+
+const TableSearchHeaderCellDiv = styled.div`
+ display: ruby;
```

Comment:
I think we don’t need this. :)

As with the top-level component, just use a `<div>` and remove `styled-components`.